networking_aci/plugins/ml2/drivers/mech_aci/cobra_manager.py

```diff
@@ -610,6 +610,12 @@ def clean_subnets(self, network):
         if network.get(az_def.AZ_HINTS):
             network_az = network[az_def.AZ_HINTS][0]
 
+        # use first not-None address scope
```

Comment:
this would take the last one from the list, if if should be the first one Ithink a break is missing.

Reply:
Yes, you're right. All not-None address scopes are the same for our subnets, but there's no need to look further once we found the first usable and the code should also be in line with the comment :)